lib/Jsrt/JsrtDebuggerObject.cpp

```diff
@@ -808,43 +822,51 @@ Js::JavascriptArray * JsrtDebugStackFrames::StackFrames(Js::ScriptContext * scri
         this->framesDictionary->Clear();
     }
 
-    uint frameCount = 0;
-
     ThreadContext* threadContext = scriptContext->GetThreadContext();
 
     DWORD_PTR dispatchHaltFrameAddress = threadContext->GetDebugManager()->GetDispatchHaltFrameAddress();
     AssertMsg(dispatchHaltFrameAddress, "Didn't set the dispatchHaltFrameAddress at time of break?");
 
-    for (Js::ScriptContext *tempScriptContext = threadContext->GetScriptContextList();
-    tempScriptContext != nullptr && tempScriptContext->IsScriptContextInDebugMode();
-        tempScriptContext = tempScriptContext->next)
+    if (dispatchHaltFrameAddress != 0)
```

Comment:
It can't be 0 for chakracore debugging, but in case it is 0 (or not set, a BUG) we won't AV and return a empty stack array.

---
In reply to: [68851362](https://github.com/Microsoft/ChakraCore/pull/1204#discussion_r68851362) [](ancestors = 68851362)